.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.3.5
```

Comment:
Support for modern multi-tool version managers, like `asdf` and `mise`.  Major IDE's also support it.